Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -56,6 +57,18 @@ async function getKubeConfigFile(): Promise<string> {
     });
 }
 
+function runHelmSaveCommand(helmCli: helmcli, kubectlCli: kubernetescli, failOnStderr: boolean): void {
+    runHelm(helmCli, "saveChart", kubectlCli, failOnStderr);
```

Comment:
Maybe we should keep a check on the version before this? If it's Helm 2, we can show appropriate message to the user.

Reply:
added the check.